src/test/resources/com/puppycrawl/tools/checkstyle/checks/javadoc/invalidjavadocposition/InputInvalidJavadocPosition.java

```diff
@@ -65,4 +65,26 @@ void method() {
         /** violation */ @Deprecated int variable3; // violation
     }
 }
+class GenericConstructor {
+    /** valid */
+    <E extends String> GenericConstructor(E a) {}
+
+    </** violation */E extends String>  GenericConstructor(E a, E b) {} // violation
```

Comment:
Please add message to trailing comment, sorry that not all inputs migrated to such model

Reply:
Done!

Reply:
drone/pr should pass if you restart it, I think